index.html

```diff
@@ -0,0 +1,131 @@
+				Tu navegador no soporta la reproducción de videos.
+			</video>
+			<div class="video-info">
+				<h1>My First Video Game With Unreal</h1>
+				<p>PapitaMC - 1M views - 1 month ago</p>
+				<div class="video-actions">
+					<button>Me gusta</button>
+					<button>No me gusta</button>
+					<button>Compartir</button>
+					<button>Agregar a</button>
+				</div>
+			</div>
+			
+			
+			<div class="video-description">
+				<p>3,307 vistas  Se estrenó el 2 jul 2022
+				Les traigo mi primer video juego realizado con Unreal, no fue creado para nada en un dia! este juego llevo bastante mastiempo esfuerzo y dedicacion, por ahora solo tengo un nivel terminado, pero pronto estara disponible su actualizacion connuevos nieveles, personajes, jefe finales Ect etc, espero les guste y puedan apoyar compartiendo este video y dejando suscomentartios y likes, gracias!  </p>
+					
+				<div class="comments">
+					<h3>Comentarios</h3>
+					<form>
+						<textarea placeholder="Escribe un comentario"></textarea>
+						<button type="submit">Publicar</button>
+					</form>
+					<div class="comments-list">
+						<div class="comment">
+							<h4>SrBLANQUITO
+								hace 9 meses</h4>
+							<p>Papita, no se si estas leyendo esto pero eres y seras siempre un ídolo de mi infancia te quiero mucho dios te bendiga y gracias por todas las alegrías que me diste Saludos.</p>
+						</div>
+		
+					</div>
+				</div>
+			</div>		
+		</div>
+
+		<div class="related-videos">
+			<h2>Related Videos</h2>
+			<ul>
+				<li>
+					<a href="#">
+						<img src="./Imagenes/min01.PNG">
+						<p>Título del video</p>
+						<p>Nombre del canal - 1M vistas - hace 1 semana</p>
+					</a>
+				</li>
+				<li>
+					<a href="#">
+						<img src="./Imagenes/min02.PNG">
+						<p>Título del video</p>
+						<p>Nombre del canal - 1M vistas - hace 1 semana</p>
+					</a>
+				</li>
+				<li>
+					<a href="#">
+						<img src="./Imagenes/min03.PNG">
+						<p>Título del video</p>
+						<p>Nombre del canal - 1M vistas - hace 1 semana</p>
+					</a>
+				</li>
+				<li>
+					<a href="#">
+						<img src="./Imagenes/min04.png">
+						<p>Título del video</p>
+						<p>Nombre del canal - 1M vistas - hace 1 semana</p>
+					</a>
+				</li>
+				<li>
+					<a href="#">
+						<img src="./Imagenes/min01.PNG">
+						<p>Título del video</p>
+						<p>Nombre del canal - 1M vistas - hace 1 semana</p>
+					</a>
+				</li>
+				<li>
+					<a href="#">
+						<img src="./Imagenes/min02.PNG">
+						<p>Título del video</p>
+						<p>Nombre del canal - 1M vistas - hace 1 semana</p>
+					</a>
+				</li>
+				<li>
+					<a href="#">
+						<img src="./Imagenes/min03.PNG">
+						<p>Título del video</p>
+						<p>Nombre del canal - 1M vistas - hace 1 semana</p>
+					</a>
+				</li>
+				<li>
+					<a href="#">
+						<img src="./Imagenes/min04.png">
+						<p>Título del video</p>
+						<p>Nombre del canal - 1M vistas - hace 1 semana</p>
+					</a>
+				</li>
+				
+			</ul>
+		</div>
```

Comment:
I suggest that you use "Section" tags instead of "div" to wrap your sections of code, as this tag is more semantic and helps to give more information about the structure of the web page.